docs/internals/release-process.txt

```diff
@@ -7,15 +7,17 @@ Django's release process
 Official releases
 =================
 
-Since version 1.0, Django's release numbering works as follows:
+Starting with version 1.0, Django's release numbering works as follows:
 
 * Versions are numbered in the form ``A.B`` or ``A.B.C``.
 
+TODO: should we replace usage of "major version" with "feature release" here?
```

Comment:
I don't agree; I think we should avoid using the terms "major" and "minor" in a way that directly conflicts with semver (even if that means we avoid using them altogether, to avoid conflicting with past Django usage). So I'd be in favor of simply replacing "major" with "feature".